test/external/kickstart_syntax_test.rb

```diff
@@ -3,10 +3,12 @@
 
 class KickstartSyntaxTest < ActiveSupport::TestCase
   ksfiles = Dir.glob('test/unit/foreman/renderer/snapshots/ProvisioningTemplate/provision/*Kickstart*')
-  ksfiles_rhel9 = ksfiles.select { |ks| ks.match?('rhel9') }
-  ksfiles_rhel7 = ksfiles - ksfiles_rhel9
 
-  versions = {'RHEL7' => ksfiles_rhel7, 'RHEL9' => ksfiles_rhel9}
+  ksfiles_v7 = ksfiles.select { |ks| ks.include?('host') }
+  ksfiles_v8 = ksfiles.select { |ks| ks.include?('rocky8') || ks.include?('rhel8') }
```

Comment:
```suggestion
  ksfiles_v8 = ksfiles.select { |ks| ks.include?('rhel8') || ks.include?('rocky8') }
```
otherwise it's confusing compared to next line :)